tests/test_classifier/test_prcurve.py

```diff
@@ -238,6 +238,44 @@ def test_multiclass_probability(self):
         tol = 6.6 if sys.platform == 'win32' else 1.0 # fails with RMSE 6.583 on AppVeyor
         self.assert_images_similar(oz, tol=tol)
 
+    def test_multiclass_probability_with_class_labels(self):
+        """
+        Visual similarity of multiclass classifier with predict_proba function
+        with class labels
+        """
+        # Create and fit the visualizer
+        oz = PrecisionRecallCurve(
+            GaussianNB(), per_class=True, micro=False, fill_area=False,
+            iso_f1_curves=True, ap_score=False,
+            classes=["a", "b", "c", "d", "e", "f"]
```

Comment:
Ok, so it's been a while since I've looked at this bug report and I just want to check to make sure I'm on the same page. By adding in this `classes=` argument, we're expecting the lines to be drawn, which they were not previously. The bug did not affect the legend correct? 

The reason I ask is that in the comment I make about the tolerance below, I suggest removing `finalize()`, which would remove the legend which would not test case of the legend being correctly set. Alternatives would be to directly test the legend object without assert image similarity, but that could be tricky; or to simply increase the tolerance so that the legend was tested. However, if the legend was part of the bug and removing finalize solves things, then perhaps we should just go that route.

Reply:
The only changes to the legend are the values of the area. 

I think at the minimum we should test the text in the legend because we want to make sure that `classes` and the calculated areas in the legend are correct.

Let me try to test for the text in the legend before increasing the tolerance.